policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_min_head_spouse_earned.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class hi_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse for the CDCC"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=29"
+        "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=41"
+        "https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2"
+    )
+
+    def formula(tax_unit, period, parameters):
+        # Schedule X PART II:
+        # line 8 & line 9
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        head_or_spouse = head | spouse
+        # Head or spouse are eligible for an income floor if disabled or a student
+        income_floor_eligible = head_or_spouse & (
+            person("is_disabled", period)
+            | person("is_full_time_student", period)
+        )
+        income = person("earned_income", period)
+        increased_income = person("hi_eligible_income_floor", period)
+        uncapped_income = where(
+            income_floor_eligible,
+            increased_income,
+            income,
+        )
+        # remove impact of income for dependents
+        head_spouse_income = where(
+            head_or_spouse, uncapped_income, tax_unit.max(uncapped_income)
```

Comment:
the `tax_unit.max(uncapped_income)` part confuses me - i think the person-level variable here will make it clearer

Reply:
I changed the `tax_unit.max(uncapped_income)` to `np.inf` now, which I think would be clearer, as we want to avoid the impact of unrelated income when finding the minimum.